src/NuGet.Core/NuGet.Frameworks/NuGet.Frameworks.csproj

```diff
@@ -4,7 +4,8 @@
 
   <PropertyGroup>
     <Description>The understanding of target frameworks for NuGet.Packaging.</Description>
-    <TargetFrameworks>$(TargetFrameworksLibrary);net40</TargetFrameworks>
+    <TargetFrameworks>$(TargetFrameworksLibrary)</TargetFrameworks>
+    <TargetFrameworks Condition="'$(DotNetBuildFromSource)' != 'true'">$(TargetFrameworks);net40</TargetFrameworks>
```

Comment:
Same here. 
https://github.com/NuGet/NuGet.Client/blob/502505da868f79b1c4fc1704dae6642138adb30c/build/common.project.props#L45

Our builds will be nicer when https://github.com/NuGet/Home/issues/8416 is finished.